tests/phpunit/multisite.xml

```diff
@@ -13,12 +13,11 @@
 		>
 	<php>
 		<const name="WP_TESTS_MULTISITE" value="1" />
-		<const name="WP_RUN_CORE_TESTS" value="1" />
```

Comment:
Noticed this was in the configuration when it shouldn't be. Just removing it while I'm here.